README.rst

```diff
@@ -1,6 +1,5 @@
-======
-Scrapy
-======
+.. image:: http://www.preserveware.com/wp-content/uploads/2017/06/scrapylogo1.png
+    :target: https://scrapy.org
```

Comment:
The logo in README looks fine to me, but I'm not comfortable serving an image from a third-party location we don't have control over.

Reply:
Good point.
I've changed it  so that it will use the [logo that is in the repo](https://github.com/scrapy/scrapy/blob/master/artwork/scrapy-logo.jpg)